README.md

```
@@ -12,6 +12,9 @@ The tool’s functionality is much broader nowadays: it can be used whenever ins
 
 [More info ==>](https://www.kontur.io/portfolio/disaster-ninja/)
 
+📖 [AI-generated tutorial](docs/toc.md)
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)